test/units/TEST-75-RESOLVED.sh

```diff
@@ -1009,6 +1011,11 @@ testcase_12_resolvectl2() {
 
 # Test io.systemd.Resolve.Monitor.SubscribeDNSConfiguration
 testcase_13_varlink_subscribe_dns_configuration() {
+    # FIXME: for some reasons, the test case unexpectedly fail when running on sanitizers.
```

Comment:
Given that these days actual bugs get hidden to make the upstream CI look green I think it would be great to be more specific here (and maybe open a separate issue to make it clear that the test isn't fully run under ASan/UBsan).

Reply:
I updated https://github.com/systemd/systemd/issues/35492 to make it clear that it isn't fully run so in terms of keeping track of that it should probably be enough. I also added the unsymbolized backtraces (unrealted to ASan) there.

Reply:
Thank you for filing that one. I do not want to tackle the issue anymore in this PR. Let's try later.